### ISSUE TITLE
"Smuggler's Den, Part 1" typo(?) on lines 136

### ISSUE THREAD

**Issue by Bladewood** [endless-sky/endless-sky#1734]
https://github.com/endless-sky/endless-sky/blob/master/data/transport%20missions.txt#L134

> You ask, "Are you in some sort of trouble with the law?"
> The boy grins. "No, not at all. I mean, aside from the fact that we're pirates."
> Seeing the incredulous look on your face, **she** says, ...

Should this say "the girl" or should it say "he"?

**Comment by retrue** [comment]
No typo. The captain is talking to Joe and Maria. Joe grins and Maria explains.

**Comment by Alkallid** [comment]
But the antecedent would be directed towards the last mentioned subject.

**Comment by endless-sky** [comment]
That was the very first mission text I wrote, for testing the conversation system back before missions even existed. The whole thing is way too wordy, but I made a first attempt at cleaning it up in commit 7ababe1.
